README.md

```diff
@@ -0,0 +1,46 @@
+OLM Manifests
+=============
+
+Integration point for all the components of KubeVirt's OLM related manifests.
+
+# Cluster Requirements
+
+The [`registry`](registry/) directory makes use of OLM's bundle support that is
+provided only via the `grpc` type `CatalogSource`. You can see us create this
+`CatalogSource` in the provided [`playbook.yaml`](playbook.yaml). This means
+that you **must** have a relatively new **OpenShift 4.0** cluster.
+
+# Ansible Requirements
+
+If you want to run the playbook you will need a recent version of Ansible (`>=
+2.6`) and the [`openshift`
+client](https://github.com/openshift/openshift-restclient-python/).
+
+# Example Run
+
+1. Get a running OpenShift 4.0 Cluster.
+1. Build the registry image: `docker build -t docker.io/djzager/kubevirt-operators:$(git rev-parse --short HEAD) -f Dockerfile .`
+1. Push the image: `docker push docker.io/djzager/kubevirt-operators:$(git rev-parse --short HEAD)`
+1. Run the Playbook `ansible-playbook playbook.yaml -e registry_image=$(git rev-parse --short HEAD)`
```

Comment:
@djzager , Even after I have installed OpenShift Python client, I see

`TASK [Kubevirt namespace state=present]`
`fatal: [localhost]: FAILED! => {"changed": false, "msg": "This module requires the OpenShift Python client. Try `pip install openshift`"}`

advise please.